core/src/test/java/org/mapfish/print/attribute/ScalebarAttributeTest.java

```diff
@@ -90,7 +90,7 @@ public void testAttributesFromJson() throws Exception {
         assertEquals("m", value.unit);
         assertTrue(value.geodetic);
         assertTrue(value.lockUnits);
-        assertEquals("Arial", value.font);
+        assertEquals("Liberation Sans", value.font);
```

Comment:
Is this font available on all systems? Should we include it in MFP so that it is always available when running the tests?